components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/EntitlementAdminService.java

```diff
@@ -539,7 +539,7 @@ public String getGlobalPolicyAlgorithm() throws EntitlementException {
      */
     public void setGlobalPolicyAlgorithm(String policyCombiningAlgorithm) throws EntitlementException {
 
-        EntitlementAdminEngine.getInstance().
-                getPolicyDataStore().setGlobalPolicyAlgorithm(policyCombiningAlgorithm);
+        ConfigDAO config = new RegistryConfigDAOImpl();
```

Comment:
This one also wont be needed with https://github.com/wso2/carbon-identity-framework/pull/5681/commits/517da62e5e80ca7376373e275f4e6797af1af331#diff-406fe136e4a3478e44cd1f6c4b1ecfaa230c904a350077d69b5e07b7861d1c54